data/provider/src/main/java/tech/pegasys/artemis/api/ChainDataProvider.java

```diff
@@ -59,6 +61,29 @@ public ChainDataProvider(
     this.chainStorageClient = chainStorageClient;
   }
 
+  public static ValidatorStatus getValidatorStatus(Validator validator, UnsignedLong epoch) {
```

Comment:
probably want to check validator and return unknown if null before we start this whole process..

Reply:
done